pkg/ccl/changefeedccl/helpers_test.go

```diff
@@ -1582,8 +1582,12 @@ func getTestingEnrichedSourceData() enrichedSourceData {
 
 // getTestingEnrichedSourceProvider creates an enrichedSourceProvider
 // for use in tests.
-func getTestingEnrichedSourceProvider(opts changefeedbase.EncodingOptions) *enrichedSourceProvider {
-	return newEnrichedSourceProvider(opts, getTestingEnrichedSourceData())
+func getTestingEnrichedSourceProvider(
```

Comment:
nit: would prefer the name of this function began with `new` instead of `get` given it's creating a new one instead of fetching one that already exists

Reply:
i dont disagree but dont want to change it here, since i didnt introduce this method and dont want to inflate the pr even more